projects/driver_controls/inc/magnetic_brake_event_generator.h

```diff
@@ -0,0 +1,28 @@
+#pragma once
+
+#include "adc.h"
+#include "ads1015.h"
+#include "ads1015_def.h"
+
+typedef struct MagneticBrakeSettings {
+  int16_t percentage_threshold;  // value above which the brake_pressed event is raised
+  int16_t zero_value;            // lsb value obtained from ads1015 when brake is unpressed
+  int16_t hundred_value;         // lsb value obtained from ads105 when brake is pressed
+  int16_t min_allowed_range;     // min allowed percentage value, usually 0
```

Comment:
There's no need to make these variables - it'll always be 0 to `EE_DRIVE_OUTPUT_DENOMINATOR`.